packages/bsky/tests/views/profile.test.ts

```diff
@@ -207,61 +207,15 @@ describe('pds profile views', () => {
     await expect(promise).rejects.toThrow('Account has been taken down')
 
     // Cleanup
-    await agent.api.com.atproto.admin.emitModerationEvent(
+    await agent.api.com.atproto.admin.updateSubjectStatus(
       {
-        event: { $type: 'com.atproto.admin.defs#modEventReverseTakedown' },
         subject: {
           $type: 'com.atproto.admin.defs#repoRef',
           did: alice,
         },
-        createdBy: 'did:example:admin',
-        reason: 'Y',
-      },
-      {
-        encoding: 'application/json',
-        headers: network.pds.adminAuthHeaders(),
-      },
-    )
-  })
-
-  it('blocked by actor suspension', async () => {
```

Comment:
If we fix the suspension functionality using the ref convention, this is just a placeholder to bring this test back.

Reply:
done 👍 & added test back in